src/models/tournament/services/seeks.service.ts

```diff
@@ -0,0 +1,117 @@
+import { HttpService } from '@nestjs/axios';
+import { HttpException, HttpStatus, Injectable, Logger } from '@nestjs/common';
+import { URL } from 'url';
+import { Color, CreateSeekDto, SeekDto } from '../dto/seek.dto';
+import { GameService } from './game.service';
+import { ConfigService } from '@nestjs/config';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Game } from '../entities/game.entity';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class SeeksService {
+	private readonly logger = new Logger(SeeksService.name);
+
+	constructor(
+		@InjectRepository(Game)
+		private readonly games: Repository<Game>,
+		private readonly gameService: GameService,
+		private readonly httpService: HttpService,
+		private readonly configService: ConfigService,
+	) {}
+
+	private getSeekApiUrl() {
+		const baseApiUrl = this.configService.getOrThrow<string>('PLAYTAK_API_URL');
+		return new URL('/api/v1/seeks', baseApiUrl);
+	}
+
+	async getSeeks(): Promise<Array<SeekDto>> {
+
+		const response = await this.httpService.axiosRef.get<Array<SeekDto>>(
```

Comment:
personally I prefer the try catch over the old school .catch I think it's cleaner but I'm not pushing a change maybe just considering consistency across the project